src/.github/workflows/{% if ci == 'GitHub' %}test.yml{% endif %}.jinja

```diff
@@ -101,15 +101,21 @@ jobs:
           persist-credentials: false
       - name: Install addons and dependencies
         run: oca_install_addons
+      {%- if github_check_license %}
       - name: Check licenses
         run: manifestoo -d . check-licenses
+      {%- endif %}
+      {%- if github_dev_status %}
       - name: Check development status
         run: manifestoo -d . check-dev-status --default-dev-status=Beta
+      {%- endif %}
```

Comment:
Instead of disabling the check, would you make this step "continue on error" (I'm not sure of the exact syntax) ?

Reply:
I believe I've added the correct incantation for this :)